File: yes.java
///////////////////////////////////
// How to build and execute
// Version - Java 8 (required)
//
// Build:
// javac yes.java
//
// option 1 - output 'yes'
// java yes 
//
// option 2 - output argument list
// java yes argument_list


class Yes {
    public static void main(String[] args) {
        String output = "y";

        if (args.length > 0) {
            output = String.join(" ", args);
        }

        while (true) {
            System.out.println(output);
        }
    }
}

File: yes.nim
# yes nim

#[
   How to build and execute
  
   Build:
   nim compile yes.nim
  
   option 1 - output 'yes'
   ./yes
  
   option 2 - output argument list
   ./yes argument_list
]#


import os
import strutils


proc yes(): void =
  var yes = ""
  if paramCount() > 0:
    yes = commandLineParams().join(" ")
    echo(yes)
  else:
    yes = "yes"
  while true:
    echo(yes)


when isMainModule:
  yes()

